code/extensions/che-telemetry/package.nls.json

```diff
@@ -0,0 +1,4 @@
+{
+	"displayName": "Eclipse Che Telemetry Extension",
+	"description": "Eclipse Che Telemetry Extension"
```

Comment:
Let's use something more descriptive for the description, e.g. `Detects and sends the events to a backend telemetry plugin`.

Reply:
Updated to:
```
Detects and sends telemetry events to a backend telemetry plugin via the Eclipse Che API built-in extension
```